doozer/tests/backend/test_golang_builder_shipment.py

```
@@ -0,0 +1,458 @@
+                release_plan="ocp-art-golang-builder-prod-rhel9",
+                nvrs=["some-nvr"],
+                ocp_version="4.22",
+            )
+
+        self.assertIn("placeholder", result)
+        mock_project.mergerequests.create.assert_not_called()
+
+    @patch("doozerlib.backend.golang_builder_shipment.python_gitlab")
+    async def test_creates_mr_with_correct_title(self, mock_gitlab):
+        handler = self._make_handler(dry_run=False)
+        handler._gitlab_token = "test-token"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "_gitlab_token"

(S105)